native/mod_manager/mod_manager.c

```diff
@@ -829,8 +848,9 @@ static apr_status_t insert_update_contexts(mem_t *mem, char *str, int node, int
     strncpy(info.context, previous, sizeof(info.context));
     if (status != REMOVE)
         ret = insert_update_context(mem, &info);
-    else
-        remove_context(mem, &info);
+    else {
```

Comment:
I would consider adding brackets around the `it` statement so it is same as the corresponding `else`.

Reply:
I am afraid it's not possible. There is an option that enforces brackets in all cases, but I don't think it's a good solution.

Reply:
I have created https://github.com/modcluster/mod_proxy_cluster/issues/92 to review and fix those.